examples/platforms/efr32/radio.c

```diff
@@ -770,15 +773,19 @@ void efr32RadioProcess(otInstance *aInstance)
     CORE_EXIT_CRITICAL();
 }
 
-void RAILCb_RfReady(void)
+void RAILCb_RxAckTimeout(void)
 {
+    if (sIsAckRequested)
```

Comment:
Is this check necessary?  If so, should we set `sIsAckRequested` to `false` here?